src/__experimental__/components/radio-button/radio-button-group.component.js

```diff
@@ -35,6 +35,8 @@ const RadioButtonGroup = (props) => {
       ml={ ml }
       mb={ mb }
       styleOverride={ styleOverride }
+      { ...(required && { required }) }
+      { ...(ariaRequired && { 'aria-required': ariaRequired }) }
       { ...tagComponent('radiogroup', props) }
     >
       <RadioButtonGroupStyle
```

Comment:
Done, I am not using RadioButtonMapper to do this but I am adding a new property: noAsterisk on the formField component. This property doesn't show the asterisk on the label (it will be useful for another component if there is no asterisk needed when the field is required, but the field stay required)